data_steward/validation/participants/validate.py

```diff
@@ -158,10 +104,16 @@ def identify_rdr_ehr_match(client,
             no_match=consts.NO_MATCH,
             missing_rdr=consts.MISSING_RDR,
             missing_ehr=consts.MISSING_EHR,
+            rdr_street=consts.NORMALIZED_STREET.render(
+                abbreviation_tuples=street_tuples_str, street='rdr_street'),
+            ehr_street=consts.NORMALIZED_STREET.render(
+                abbreviation_tuples=street_tuples_str, street='ehr_street'),
+            rdr_city=consts.NORMALIZED_CITY.render(
+                abbreviation_tuples=city_tuples_str, city='rdr_city'),
+            ehr_city=consts.NORMALIZED_CITY.render(
+                abbreviation_tuples=city_tuples_str, city='ehr_city'),
```

Comment:
I think these could be static tables in drc_ops. They just need the '_' prefix as they are internal tables. We shouldn't need to change them often. Let me know your thoughts.